src/service/resources-impl.js

```diff
@@ -771,10 +777,16 @@ export class Resources {
           resize = true;
         } else if (diff < 0) {
           // 6. The new height is smaller than the current one.
+          if (request.callback) {
+            request.callback(false);
+          }
           resize = false;
         } else {
           // 7. Element is in viewport don't resize and try overflow callback
           // instead.
+          if (request.callback) {
+            request.callback(false);
+          }
           request.resource.overflowCallback(/* overflown */ true,
```

Comment:
Then let's move this into a catch block of the promise chain. As it is now it's extremely confusing.